contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowAppendOutputOperator.java

```diff
@@ -63,50 +55,35 @@
  *            The tuple type
  * @since 1.0.2
  */
-public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractAggregateTransactionableStoreOutputOperator<T, HBaseWindowStore> {
+public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractHBaseWindowOutputOperator<T> {
   private static final transient Logger logger = LoggerFactory.getLogger(AbstractHBaseWindowAppendOutputOperator.class);
-  private List<T> tuples;
   private transient ProcessingMode mode;
+  
+  @Deprecated
```

Comment:
Processing mode being a common platform feature should be set as an attribute so not sure why it was made available as a property. Even if the property is set, it wouldn't effect how the platform would process that feature for the operator.